translations/bnpchs-final-2018-Susu-Moli.html

```diff
@@ -0,0 +1,39 @@
+<!-- Title: Moli Milk-->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix likes recently popular street food called "Es Kepal Moli" a lot. After receiving Jelix's Es Kepal Moli, one of his friends, Fingga got curious, about the method of making Es Kepal Moli. Surprisingly one of the ingredients used by Jelix is Moli branded milk. 
```

Comment:
```suggestion
  Jelix likes the newly popular street food called <em>Es Kepal Moli</em> a lot. 
  After receiving Jelix's <em>Es Kepal Moli</em>, one of his friends, Fingga got curious
  about the method of making Es Kepal Moli. 
  Surprisingly, one of the ingredients used by Jelix is Moli branded milk. 
```

Reply:
uwu i used word wrap in vscode, always forgot to multi line